src/vs/workbench/contrib/notebook/browser/contrib/navigation/arrow.ts

```diff
@@ -225,7 +226,7 @@ registerAction2(class extends NotebookCellAction {
 	async runWithContext(accessor: ServicesAccessor, context: INotebookCellActionContext): Promise<void> {
 		const editor = context.notebookEditor;
 		const activeCell = context.cell;
-		await editor.focusNotebookCell(activeCell, 'output');
+		return timeout(0).then(() => editor.focusNotebookCell(activeCell, 'output'));
```

Comment:
nextTick or setImmediate provides this functionality: https://stackoverflow.com/a/15349865/1253265

Reply:
oh you want the promise, maybe not then